### DISCUSSION TITLE
Disable white chevron markers indicating end of file

### DISCUSSION THREAD

**Discussion by Nic-Alexander** [variar/klogg#729] [Q&A]
![KLogg](https://github.com/user-attachments/assets/79ecb415-0158-4173-9f6b-336ad5e3b100)
If you open a file by passing a file name to klogg and you have follow end enabled, a white chevron bar is displayed showing that follow end is enabled

This is a very distracting visual.   Baretail just follows the end.   Is there any way of disabling this display feature?

**Comment by variar** [comment]
No, at the momend there is no way to disable this visual indicator. Pull requests are welcome
